.azure/scripts/metadata_validator.py

```diff
@@ -10,12 +10,13 @@
 VALID_COMPONENT_TYPES = [
     "service", "webhook", "manual-task", "scheduled-task", 
     "event-triggered", "event-handler", "test-runner", "many",
-    "web-application"
+    "web-application", "git-proxy"
```

Comment:
Please add this to the `componentTypeMap` in choreo-console.
https://github.com/wso2-enterprise/choreo-console/blob/7351666de3727a38fda71d6a551bdb4a20b7f758/src/hooks/samples.tsx#L56

Reply:
Addressed : https://github.com/wso2-enterprise/choreo-console/pull/9704